oak/server/rust/oak_runtime/src/node/http/mod.rs

```diff
@@ -428,8 +428,8 @@ fn get_oak_label(req: &HttpRequest) -> Result<Label, OakStatus> {
             OakStatus::ErrInvalidArgs
         }),
         None => {
-            warn!("No HTTP label found:");
-            Err(OakStatus::ErrInvalidArgs)
+            warn!("No HTTP label found: using public_untrusted.");
+            Ok(Label::public_untrusted())
```

Comment:
This seems a very dangerous change: it is going to assume that any data sent to this server is public unless an explicit header is provided, but surely in that case Oak is not doing anything useful at all, so it's almost definitely a mistake from the client. We should fail closed in this case.

Reply:
I'll fix it with #1345.